src/Compilers/CSharp/Portable/Parser/LanguageParser.cs

```diff
@@ -8892,6 +8892,8 @@ private static Precedence GetPrecedence(SyntaxKind op)
                 case SyntaxKind.ThisExpression:
                 case SyntaxKind.TrueLiteralExpression:
                 case SyntaxKind.TupleExpression:
+                // PROTOTYPE: Is this the right precedence?
```

Comment:
>PROTOTYPE [](start = 19, length = 9)

Given the similarity in syntax, I suggest it should be at the same precedence as the switch expression.